### ISSUE TITLE
Add resistance factor?

### ISSUE THREAD

**Issue by mostaphaRoudsari** [mostaphaRoudsari/EmergeSettlement#7]
"For optimisation or land reallocation I wonder if we need to add a 'resistance' value for how easily land can be converted. Previously we had it based on a hierarchy of which was easiest to convert but in reality there is a trade off between distance from the settlement, available land and cost. Perhaps this over complicates it though."

cc: @Butcherluke

**Comment by Butcherluke** [comment]
I think this would be a good addition but may over complicate things for the initial paper write up?

**Comment by Butcherluke** [comment]
Some more thoughts on this (and thinking back to the conversation last week about how do you determine the limits from which a settlement point can be drawn). Access to 'services' (land in a pixel) is not simply about distance but the time it takes to get there: https://www.youtube.com/watch?v=u5_uC8Eif0w (about 4:52 in)

**Comment by mostaphaRoudsari** [comment]
I think we can calculate that using services like GoogleMaps for every city to every pixel.

**Comment by Butcherluke** [comment]
Yes that sounds like a good plan! It would add another dimension to the scenario builder. I gave a sneak preview today at a study session I have for my current course and the question of distance and time was one that came up.

**Comment by brahamw** [comment]
I was just going to say that transportation is the factor we have neglected...

**Comment by Butcherluke** [comment]
@brahamw the above link I posted offers one possible scenario. It also brings into question issues of growth boundaries which we could introduce as another factor of resistance to growth in version 2.0

**Comment by Butcherluke** [comment]
Alongside a 'resistance' factor we could add a slider for how far pixels can be taken from the settlement point to supply that population. There may be work done by urban or transport planners about optimum distances for this. But it would be interesting if users could define the voundary/distance limits themselves. This would force more land to be reallocated and avoid the very large 'sheds' the current model throws up.

**Comment by mostaphaRoudsari** [comment]
@Butcherluke I agree and we also need a default value for the current stage. It can also change based on population?
